pre_commit_hooks/pretty_format_json.py

```diff
@@ -90,20 +100,23 @@ def pretty_format_json(argv=None):
     status = 0
 
     for json_file in args.filenames:
-        with open(json_file) as f:
+        with io.open(json_file, encoding='utf-8') as f:
```

Comment:
I think that's a reasonable assumption, the json spec indicates they should have some UTF-x encoding